earth_enterprise/src/third_party/gdal/README.txt

```diff
@@ -35,3 +35,11 @@ on a SLES 9 box (compatible with SLES 9,10, RHEL 4,5 and Ubuntu 6,8)
 
 To Build, you will need to build on a SLES 9 box using the scripts found in
 //depot/googleclient/geo/fusion/RPMS/build_scripts
+
+To build GDAL with MrSID Raster Format, MrSID library needs to be installed on your platform:
+1- Get MrSID Decoding SDK library
```

Comment:
I happen to know where to get this library, so I'm speaking on behalf of the engineers who will be coming along after us :)

Can you add a link to the LizardTech page where you can sign up to request and download the SDK? (https://www.lizardtech.com/developer) I searched for "MrSID Decoding SDK library" and I didn't see that page come up in the first page of Google results.

Reply:
sure